clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1052,6 +989,55 @@ void Sema::MarkDevice(void) {
     }
   }
 }
+//
+// Do we know that we will eventually codegen the given function?
```

Comment:
I appreciate all these comments. Thanks. :-)

Reply:
LOL!  The patch isn't "finished".   I was mainly looking for a yay/nay on this direction for deferring diagnostics.   If that aspect looks good, I will upload a cleaner version soon.

Reply:
I mean you are participating to the implementation of https://github.com/intel/llvm/issues/189 :-)